### PR TITLE
Reverts changes using style resource for default theme

### PR DESCRIPTION
**Pull request by rstm-sf**
## What does the pull request do?
Reverts changes using style resource for default theme

## What is the current behavior?
Now, when using the default theme, resources from the fluent theme are used

## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Avaloniaui.net with user documentation

### CODE REVIEW

File: src/Avalonia.Themes.Default/Window.xaml [hunk 1]
```diff
@@ -1,9 +1,8 @@
 <Style xmlns="https://github.com/avaloniaui" Selector="Window">
-  <Setter Property="Background" Value="{DynamicResource SystemControlBackgroundAltHighBrush}"/>  
-  <Setter Property="TransparencyBackgroundFallback" Value="{DynamicResource SystemControlBackgroundAltHighBrush}" />
-  <Setter Property="Foreground" Value="{DynamicResource SystemControlForegroundBaseHighBrush}"/>
-  <Setter Property="FontSize" Value="{DynamicResource ContentControlFontSize}"/>  
-  <Setter Property="FontFamily" Value="{DynamicResource ContentControlThemeFontFamily}" />
```
Comment (rstm-sf) [inline src/Avalonia.Themes.Default/Window.xaml:6]: Do we really need this?

File: src/Avalonia.Themes.Default/TitleBar.xaml [hunk 2]
```diff
@@ -23,7 +23,7 @@
   </Style>
 
   <Style Selector="TitleBar:fullscreen">
-    <Setter Property="Background" Value="{DynamicResource SystemAccentColor}" />
+    <Setter Property="Background" Value="{DynamicResource ThemeAccentColor}" />
```
Comment (rstm-sf) [inline src/Avalonia.Themes.Default/TitleBar.xaml:26]: And I'm not sure about that either

**Comment by Gillibald** [comment]
You can't change the naming of resources that are used in the default theme. This would break existing applications and other themes that are based on the default. If something is wrong with the default it needs to be fixed. Fluent and default theme have no relation.

**Comment by rstm-sf** [comment]
These changes attempt to revert back

**Comment by maxkatz6** [comment]
> Fluent and default theme have no relation.
It's true. So that's because there shouldn't be any "System*" and "ContentControl*" resources in Default theme.
I believe they were added by mistake in last preview versions.

**Comment by rstm-sf** [comment]
See also https://github.com/AvaloniaUI/Avalonia/commit/36c5ad4621d372cf932f1259c77a23c4d1cb6962#diff-a40f6d994eb00baa9376f60fc0358058 and https://github.com/AvaloniaUI/Avalonia/commit/d8604567eec0445e2898a78437b5016b021a2cd4#diff-36cefb95cab1af4e2d37d01a10734cc7

**Comment by Gillibald** [comment]
We should probably revert this https://github.com/AvaloniaUI/Avalonia/commit/d8604567eec0445e2898a78437b5016b021a2cd4

**Review by Gillibald — APPROVED** [review]

**Comment by jmacato** [comment]
@rstm-sf please revert the default theme part of https://github.com/AvaloniaUI/Avalonia/commit/d8604567eec0445e2898a78437b5016b021a2cd4 then we can merge

**Comment by rstm-sf** [comment]
If we do this completely, then the Window Customization tab in the ControlCatalog will not work properly for the default theme

**Comment by grokys** [comment]
@rstm-sf this has changed the font size to small for the default theme, was this intended?

image

Text in devtools looks smaller than before.

**Comment by maxkatz6** [comment]
@rstm-sf @Gillibald I believe Window.FontSize should be FontSizeNormal instead of FontSizeSmall

**Comment by rstm-sf** [comment]
@grokys sorry my mistake. Installed `FontSizeNormal` instead of `FontSizeSmall`

**Pull request merged**